test/Base_Tests/src/Data/Time/Date_Time_Formatter_Spec.enso

```diff
@@ -78,7 +78,7 @@ add_specs suite_builder =
             d.format (Date_Time_Formatter.from "d. MMMM yyyy" Locale.default) . should_equal "30. June 2020"
 
         group_builder.specify "should allow using a Java formatter" <|
-            jformatter = Date_Time_Formatter.from_java DateTimeFormatter.ISO_ORDINAL_DATE
+            jformatter = Date_Time_Formatter.iso_ordinal_date
```

Comment:
- solves https://github.com/enso-org/enso/pull/13570#discussion_r2293700623
- by avoiding access to Java and using Enso API
- a useful approach when having a mismatch between "dual JVMs"